apps/src/code-studio/pd/application/principalApproval1819/PrincipalApproval1819Application.jsx

```diff
@@ -15,6 +15,19 @@ export default class PrincipalApproval1819Application extends FormController {
       principal_email: PropTypes.string.isRequired
     }).isRequired
   }
+  /**
+   * @override
+   */
+  constructor(props) {
+    super(props);
+
+    Object.assign(this.state.data, {
+      firstName: this.props.teacherApplication.principal_first_name,
+      lastName: this.props.teacherApplication.principal_last_name,
+      title: this.props.teacherApplication.principal_title,
+      email: this.props.teacherApplication.principal_email,
```

Comment:
nit: The base [React.Component](https://github.com/facebook/react/blob/master/packages/react/src/ReactBaseClasses.js#L18) class does set `this.props = props`, but still I think it would be clearer here in the constructor to refer to the local `props`